libexec/rtld-elf/cheri/cheri_reloc.h

```diff
@@ -36,26 +36,25 @@
 #include "debug.h"
 #include "rtld.h"
 
+#ifdef RTLD_HAS_CAPRELOCS
 /* The clang-provided header is not warning-clean: */
 __unused static void cheri_init_globals(void);
 #include <cheri_init_globals.h>
-#if !defined(CHERI_INIT_GLOBALS_VERSION) || CHERI_INIT_GLOBALS_VERSION < 4
+#if !defined(CHERI_INIT_GLOBALS_VERSION) || CHERI_INIT_GLOBALS_VERSION < 5
 #error "cheri_init_globals.h is outdated. Please update LLVM"
 #endif
 
-extern bool add_cheri_plt_stub(const Obj_Entry *obj, const Obj_Entry *rtldobj,
-    Elf_Word r_symndx, void **where);
-
 /* FIXME: replace this with cheri_init_globals_impl once everyone has updated clang */
```

Comment:
Maybe we should replace this with a call to `cheri_init_globals_impl` now?

Reply:
Meh, caprelocs's days are numbered for dynamic binaries, I'm somewhat reluctant to do small fixes like that, but I won't stop you if you want to inline it

Reply:
Yeah I only thought since you're changing the function anyway. No need to do it.